lib/main_common.pm

```diff
@@ -879,6 +882,7 @@ sub load_inst_tests {
             and !check_var('BACKEND', 's390x')
             and !check_var('BACKEND', 'ipmi')
             and is_sle('12-SP2+'))
+          and !check_var('BACKEND', 'spvm')
```

Comment:
But should be added after `and !check_var('BACKEND', 'ipmi')`, I think it's more readable and more important it will work, because like this it is not :-) Because of the `)` at the end of `and is_sle('12-SP2+')`.

Reply:
fixed